specification/metrics/pricingquantity.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Quantity
+
+The Pricing Quantity represents the volume of a given resource and/or service used or purchased based on the [Pricing Unit](#pricingunit). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
+
+PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [AdjustmentType](#adjustmenttype) is 'Refund'. This column MUST NOT be null if unit prices are not null and MUST be null if unit prices are null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Comment:
```suggestion
PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [AdjustmentType](#adjustmenttype) is 'Refund'. This column MUST NOT be null if [SkuPriceId](#skupriceid) is not null and MUST be null if SkuPriceId is null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Reply:
I prefer the original version as it aligns with the subsequent sentence referring to unit prices not being null...

Reply:
unit prices seems not very clear to me since there may be multiple in the future (negotiated) and also, is not referring to something that can be linked to.